met-web/src/constants/commentStatus.ts

```diff
@@ -2,4 +2,5 @@ export enum CommentStatus {
     Pending = 1,
     Approved = 2,
     Rejected = 3,
+    NeedsFurtherReview = 4,
```

Comment:
If I remember correctly I think NeedsFurtherReview = Pending, so i think we don't need a new status